src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -64,7 +70,13 @@ const getParcelsForDelivery = async (parcelIds: string[]): Promise<ParcelsForDel
             );
             return { data: null, error: { type: "noMatchingClient", logId: logId } };
         }
-        dataWithNonNullClients.push({ ...parcel, client: parcel.client });
+
+        let labelCount: number | undefined;
+        if (parcel.events && parcel.events.length > 0 && parcel.events[0].event_data) {
```

Comment:
Hope js/ts has a standard function for it but you might be able to simplify this to something like `if (!parcel.events.isNullOrEmpty() && parcel.events[0].event_data)`

Reply:
As far as I can tell there isn't a way to combine a nullish check and an empty array check built in to js/ts, so the first two conditions have to be like this. Lmk if you know of a better way! :)

Reply:
Hmm yeah doesn't look like it, you could try null coalescence and have something like `parcel.events?.length > 0 && ...` or `parcel.events && parcel.events[0]?.event_data` but I don't think it improves much - if anything it makes it more confusing for the sake of being shorter